### DISCUSSION TITLE
Links on console are using default text color

### DISCUSSION THREAD

**Discussion by samarthpuri** [twilio-labs/paste#3098] [General] [answered]
Hey Paste team! 

I've been noticing for a while now that the links in Console are using the default text color instead of the usual blue. Attached some screenshots as examples 

<img width="411" alt="Screen Shot 2023-03-15 at 2 47 36 PM" src="https://user-images.githubusercontent.com/23617133/225438611-fdb6edcb-bc81-4d8d-92d1-1b242e3c3081.png">
<img width="577" alt="Screen Shot 2023-03-15 at 2 48 27 PM" src="https://user-images.githubusercontent.com/23617133/225438810-39fa977e-f9a3-41d7-ab84-42a91f88f287.png">
<img width="279" alt="Screen Shot 2023-03-15 at 2 49 19 PM" src="https://user-images.githubusercontent.com/23617133/225439021-5ae5c691-10f6-4f17-bfd1-8b2246930dc2.png">
<img width="637" alt="Screen Shot 2023-03-15 at 2 50 08 PM" src="https://user-images.githubusercontent.com/23617133/225439179-23194ad5-4634-418c-b5fe-ec0c6df003a8.png">

**Comment by SiTaggart** [comment, marked as answer]
Hi there

As per my response to the issue created about the same topic, this is an issue with the Console app.

https://github.com/twilio-labs/paste/issues/3099
